backend/de.metas.acct.webui/src/main/java/de/metas/acct/gljournal_sap/quickinput/SAPGLJournalLineQuickInputDescriptorFactory.java

```diff
@@ -152,6 +159,33 @@ private LookupDescriptorProvider getPostingSignLookup()
 		return lookupDescriptorProviders.listByAD_Reference_Value_ID(postingSignReferenceId);
 	}
 
+	@NonNull
+	private LookupDescriptorProvider getAccountProvider()
+	{
+		final POInfo poInfo = POInfo.getPOInfo(I_SAP_GLJournalLine.Table_Name);
+		if(poInfo == null){return getAccountProviderFallback();}
+
+		final POInfoColumn columnInfo = poInfo.getColumn(I_SAP_GLJournalLine.COLUMNNAME_C_ValidCombination_ID);
+		if(columnInfo == null){return getAccountProviderFallback();}
```

Comment:
imho this shall not happen... or to put it differently, if that's happening, this quick input is pointless!